modules/mesaides/files/ma-web.conf

```diff
@@ -11,4 +11,11 @@ setuid main
 
 chdir /home/main/mes-aides-ui
 
-exec node server
+script
+    set -a
+    if [ -f /home/main/.secrets ]
```

Comment:
Use a more specific name. I agree we can assume to be alone on the machine, but still, a `.mes-aides-secrets` would make forensics much easier :wink: